Cartfile.resolved

```diff
@@ -1,3 +1,3 @@
-binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "3.4.2"
-github "AliSoftware/OHHTTPStubs" "57feceaabf333e72b2c637dfba6c13a7a5c49619"
+binary "https://www.mapbox.com/ios-sdk/Mapbox-iOS-SDK.json" "3.5.0"
+github "AliSoftware/OHHTTPStubs" "f90c2bb0fb882e43761ab963ca8869d349d2c6e3"
```

Comment:
This is fine, but in the future, let’s split out any changes that occur due to running `carthage update` into separate PRs, so that we can narrow down any regressions they cause. If you just need to get set up, use `carthage bootstrap` instead.